### ISSUE TITLE
[Bug]:[a11y]: Talkback doesn't read the checkbox states from the "Delete browsing data" screen

### ISSUE THREAD

**Issue by AdinaPetridean** [mozilla-mobile/fenix#26435]
### Steps to reproduce

1. Turn TalkBack (or the available screen reader) on.
2. Go to 3 dot menu.
3. Enter in the Settings screen.
4. Go to the "Delete browsing data" screen.
5. Select any of the options displayed. Observe the behavior.

### Expected behaviour

Talkback reads the state of the selected checkbox. 

### Actual behaviour

Talkback doesn't read the state of the selected checkbox.

### Device name

Xiaomi Pad 5

### Android version

Android 11

### Firefox release type

Firefox Beta

### Firefox version

Firefox Beta 104.0b5

### Additional information

![1660218964926](https://user-images.githubusercontent.com/108326843/184128411-8884ceb1-82c7-42b0-bc26-d68cf460db9e.JPEG)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21273)

**Comment by lobontiumira** [comment]
I was not able to reproduce this issue on the Beta 105.0b3 build with Google Pixel (Android 10).

**Comment by AdinaPetridean** [comment]
This issue is not reproducible on Beta 106.0b2 build with Xiaomi Pad 5 (Android 12).
